### DISCUSSION TITLE
Stop trying to be Twitter!!!!

### DISCUSSION THREAD

**Discussion by TeddAndersen** [community/community#55023] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If i wanted a page full of to me useless stuff i would like this change... but i dont!!

Stop trying to be twitter....
Let us turn off/dissable/remove this useles "For you"

**Comment by fmaida** [comment]
I could not agree more. Please remove this garbage from the dashboard.

**Comment by cristoffer** [comment]
Yes, let us opt out of this. 
I don't see how this social media experiment aligns with the GitHub platform, its purpose and use.

**Comment by dnut** [comment]
I go to a homepage hoping for quick access my commonly used resources. A list of pull requests I committed to recently would be ideal. A list of repositories I have committed to recently would also be nice, especially if the homepage also included links to issues & PRs for each of those repos, so I don't have to load multiple pages on my way there.

But no, none of that exists. Instead the homepage is a chaotic display of noise that is completely irrelevant to anything I am working on. I get a glimmer of hope when I see this surprisingly small "top repositories" section with a few projects, but it doesn't include any of the repositories I am active in on a daily basis, and it is not customizable in any way. It includes inactive repositories that I only committed to a single time, over a year ago. What the hell? The "Recent activity" section is another disappointment. It includes things I have never contributed to, and excludes prs that I created, plus it mixes content that is too diverse: both issues and prs from all repos. These should be separate, larger sections. It ends up being too noisy and rarely has the content I need, so I don't use it.

The only useful thing on the github.com homepage is the search bar, and we all know how well that works. I can type the exact name of a repository that I own and it doesn't even show up in the first page of results.
